### DISCUSSION TITLE
mockCall on function with no return?

### DISCUSSION THREAD

**Discussion by 0xend** [foundry-rs/foundry#1884] [Help]
I'm trying to mock a call to a function that does not return any value but I can't get it to work.

If I modified the function to return something, then things would work fine. I'm not sure if there's a way to call mockCall such that it performs as expected in these cases (tried `bytes("")` to no success) or I should be using a completely different cheatcode.

Thanks!

**Comment by onbjerg** [comment]
Hi! What is your use case? The primary intended use case for `mockCall` is to mock return data, and I don't immediately see the benefit of trying to mock a call to something that doesn't return anything - can you clarify?
